indexers/vector/FaissIndexer/__init__.py

```diff
@@ -106,19 +109,23 @@ def build_advanced_index(self, vecs: 'np.ndarray'):
                     faiss.normalize_L2(train_data)
                 self._train(index, train_data)
 
-        self.build_partial_index(vecs, index)
+        self._build_partial_index(vecs, index)
         index.nprobe = self.nprobe
         return index
 
     @batching
-    def build_partial_index(self, vecs: 'np.ndarray', index):
+    def _build_partial_index(self, vecs: 'np.ndarray', index):
         vecs = vecs.astype(np.float32)
         if self.normalize:
             from faiss import normalize_L2
             normalize_L2(vecs)
         index.add(vecs)
 
     def query(self, vecs: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple['np.ndarray', 'np.ndarray']:
+        """Find the top-k vectors with smallest ``metric`` and return their ids in ascending order.
```

Comment:
this has an exception for metric inner_product, logic should be changed or warning added